swaggen/src/main/java/generator/PathGenerator.java

```diff
@@ -19,15 +19,16 @@
 	/**
 	 * Generates a Map of URLs and RequestMethods to Endpoints.
 	 * 
-	 * @param klasses s list of annotated classes
+	 * @param klasses s set of annotated classes

```

Comment:
Incorrect. It is an "array of annotated classes." Correct semantics if you must, but do so correctly.